coderd/users.go

```diff
@@ -1343,6 +1347,10 @@ func (api *API) CreateUser(ctx context.Context, store database.Store, req Create
 	err := store.InTx(func(tx database.Store) error {
 		orgRoles := make([]string, 0)
 
+		status := ""
+		if req.UserStatus != nil {
+			status = string(*req.UserStatus)
+		}
```

Comment:
similarly when would we expect the status to be empty?

Reply:
Empty just means use the default behavior, which is `dormant`. It would be a breaking change if this was required